src/backend/defs/rest-api/swagger-root.json

```diff
@@ -19,7 +19,9 @@
     "/discussions": {"$ref": "paths/discussions.json#/discussions"},
     "/discussions/{discussion_id}": {"$ref": "paths/discussions.json#/discussions_{discussion_id}"},
     "/discussions/{discussion_id}/messages": {"$ref": "paths/discussions.json#/discussions_{discussion_id}_messages"},
-    "/raws/{raw_msg_id}": {"$ref": "paths/raws.json#/raws_{raw_msg_id}"}
+    "/raws/{raw_msg_id}": {"$ref": "paths/raws.json#/raws_{raw_msg_id}"},
+    "/tags": {"$ref": "paths/tags.json#/tags"},
+    "/tags/{name}": {"$ref": "paths/tags.json#/tags_{name}"}
```

Comment:
Entities will produce list of associated tags, but they can produce only name of them or complete tag structure.

['important', 'boulot']

[{'name': 'important', 'type': 'system', 'label': 'Important', ....]

The third point is a search in some entities having this tag.

Remark: a tag on an entity may does not exist anymore in the current list on GET /tags. Managing a list of current tags and have historical tag values differents is 2 distinct things